src/reactor.rs

```diff
@@ -502,7 +509,7 @@ impl<T> Future for Readable<'_, T> {
 
     fn poll(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output> {
         ready!(Pin::new(&mut self.0).poll(cx))?;
-        log::trace!("readable: fd={:?}", self.0.handle.source.registration);
+        tracing::trace!(fd=?self.0.handle.source.registration, "readable");
```

Comment:
```suggestion
        tracing::trace!(fd = ?self.0.handle.source.registration, "readable");
```

(It may be a bit easier to read if there is a space before and after the `=`.)